incubator/hnc/test/e2e/demo_test.go

```diff
@@ -76,7 +76,7 @@ var _ = Describe("Demo", func() {
 
 	It("Should propagate different types", func() {
 		// ignore Secret in case this demo is run twice and the secret has been set to 'Propagate'
-		MustRun("kubectl hns config set-type --apiVersion v1 --kind Secret Ignore")
+		MustRun("kubectl hns config set-type --resource secrets Ignore")
```

Comment:
File bug to fix documentation?

Reply:
Filed #1207 under #868 . Won't add `todo` in any code for now since it's well tracked and not related to code any more.